templates/keras_cv/index.md

```diff
@@ -2,87 +2,219 @@
 
 <a class="github-button" href="https://github.com/keras-team/keras-cv" data-size="large" data-show-count="true" aria-label="Star keras-team/keras-cv on GitHub">Star</a>
 
-KerasCV is a toolbox of modular building blocks (layers, metrics, losses, data augmentation) that computer vision engineers can leverage to quickly assemble production-grade, state-of-the-art training and inference pipelines for common use cases such as image classification, object detection, image segmentation, image data augmentation, etc.
+KerasCV is a library of modular computer vision oriented Keras components.
+These components include models, layers, metrics, losses, callbacks, and utility
+functions.
+
+KerasCV's primary goal is to provide a coherent, elegant, and pleasant API to
+train state of the art computer vision models. Users should be able to train
+state of the art models using only `Keras`, `KerasCV`, and TensorFlow core (i.e.
+`tf.data`) components.
+
+KerasCV can be understood as a horizontal extension of the Keras API: the
+components are new first-party Keras objects (layers, metrics, etc.) that are
+too specialized to be added to core Keras. They receive the same level of polish
+and backwards compatibility guarantees as the core Keras API, and they are
+maintained by the Keras team.
+
+Our APIs assist in common computer vision tasks such as data-augmentation,
+classification, object detection, image generation, and more. Applied computer
+vision engineers can leverage KerasCV to quickly assemble production-grade,
+state-of-the-art training and inference pipelines for all of these common tasks.
+
+In addition to API consistency, KerasCV components aim to be mixed-precision
+compatible, QAT compatible, XLA compilable, and TPU compatible. We also aim to
+provide generic model optimization tools for deployment on devices such as
+onboard GPUs, mobile, and edge chips.
+
+## Quick Links
+- [List of available models and presets](https://keras.io/api/keras_cv/models/)
+- [Developer Guides](https://keras.io/guides/keras_cv/)
+- [Contributing Guide](.github/CONTRIBUTING.md)
+- [Call for Contributions](https://github.com/keras-team/keras-cv/issues?q=is%3Aopen+is%3Aissue+label%3Acontribution-welcome)
+- [API Design Guidelines](.github/API_DESIGN.md)
 
-<img style="width: 440px; max-width: 90%;" src="/img/keras-cv-augmentations.gif">
```

Comment:
I really like this gif, let's keep it!!!

Reply:
Done.